state/tfe.go

```diff
@@ -0,0 +1,169 @@
+package state
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/camptocamp/terraboard/config"
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/states/statefile"
+	//log "github.com/sirupsen/logrus"
```

Comment:
If it's unused, you might as well remove it

Reply:
indeed 🤦‍♂ 👍